package.json

```diff
@@ -1,24 +1,15 @@
 {
   "name": "compute-starter-kit-javascript-empty",
-  "version": "0.3.0",
+  "version": "0.4.0",
   "type": "module",
   "main": "src/index.js",
   "author": "oss@fastly.com",
   "license": "MIT",
   "engines": {
-    "node": ">=16.0.0"
-  },
-  "devDependencies": {
-    "core-js": "^3.19.1",
-    "webpack": "^5.64.0",
-    "webpack-cli": "^4.9.1"
+    "node": ">=18.0.0"
   },
   "dependencies": {
-    "@fastly/expressly": "github:fastly/expressly",
-    "@fastly/js-compute": "^1.1.0"
-  },
-  "scripts": {
```

Comment:
Is this convention now? The default starter kit still has these scripts.

Reply:
It's not convention, but the default starter is much more generic and here the scripts aren't used anywhere apart from CI. The starter didn't come with a webpack config from the beginning and we don't _need_ webpack, so I think in this instance it is more confusing to keep the scripts and devDependencies around.